tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Create User Validation.3540.feature

```diff
@@ -0,0 +1,118 @@
+﻿Feature: Create Buyer Account Validation
+    As an Authority User
+    I want to see errors when creating a new user
+    So that I know when something has gone wrong
+
+Background:
+	Given Organisations exist
+		| Name           | OdsCode |
+		| Organisation 1 | Ods 1   |
+		| Organisation 2 | Ods 2   |
+	And Users exist
+		| Id     | OrganisationName | FirstName | LastName | Email                | PhoneNumber | Disabled | Password        | OrganisationFunction |
+		| 012345 | Organisation 1   | Penny     | Lane     | PennyLane@email.com  | 01234567890 | false    | S0mePa$$w0rd    | Buyer                |
+		| 123456 | Organisation 1   | Post      | Pat      | PostmanPat@email.com | 12345678901 | false    | An0therPa$$w0rd | Authority            |
+	And a user is logged in
+		| Username             | Password        | Scope        |
+		| PostmanPat@email.com | An0therPa$$w0rd | Organisation |
+
+@3540
+Scenario: Empty first name yields a first name is required error
```

Comment:
Are you not wanting to have numbers for each test?

Reply:
I'm not sure they provide any value. We don't do it for the unit tests. However I agree for consistency reasons I will put them back in. To be discussed in the next coding standards meeting.